### ISSUE TITLE
Add navigation for modified files

### ISSUE THREAD

**Issue by stinos** [sublimehq/sublime_merge#961]
**Problem description**

When working with more than a couple of modified files and wanting to select only some of them for reviewing/staging it gets hard to find specific files in the staging/diff (a.k.a. 'Summary') view. One first has to collapse the Working Directory then hunt for the file in the alphabetic list. This is also hinted a bit and/or slightly related to e.g. #418 #624 #931.

**Preferred solution**

The now sort of canonical way for navigating to a file is with fuzzy matching i.e. Ctrl-P (which was actually the first thing I tried): a keyboard shortcut which opens a view with a list of unstaged and untracked files from which one can be selected, followed by bringing the diff of that file into view.

**Alternatives**

Looked around a bit but found no way to jump to the changes of a specific file in the diff.

**Comment by dpjohnst** [comment]
Hi @stinos,

Thank you for sharing this feedback. The current recommended way to navigate large commits is to use the "Files" tab. I understand though that even with extremely large diffs, this can take time to navigate.

The idea of fuzzy-matching / filtering diffs is definitely useful, and this is something the team will be exploring.

Thanks,
\- Dylan

**Comment by stinos** [comment]
> the "Files" tab. I understand though that even with extremely large diffs, this can take time to navigate.

Exactly, moreover apart from consuming less screen height the 'Files' tab isn't different from the diff view with everything collapsed.
